sample-apps/README.md

```diff
@@ -38,3 +38,245 @@ List of sample apps across all repositories in [aws-observability](https://githu
+
+**Request based:**
+
+AsyncCounter
+
+* Name: totalApiRequests
+* Unit: “1”
+* Description: “Increments by one every time a sampleapp endpoint is used”
+
+Counter
+
+* Name: totalBytesSent
+* Unit: “By”
+* Description: “Keeps a sum of the total amount of bytes sent while the application is alive”
+* Includes Callback Function
+
+Histogram
+
+* Name: latencyTime
+* Unit: “ms”
+* Description: “Measures latency time in buckets of 100 300 and 500”
+
+**Common Attributes for Request based metrics**
+
+These are Key Value pairs to be added on metrics.
+```
+{
+“signal”:     (string)“metric”
+“language”:   (string)<name of language used>
+“metricType”: (string)“request”
+}
+```
+
+**Random:**
+
+Counter
+
+* Name: timeAlive
+* Unit: “ms”
+* Description: “Total amount of time that the application has been alive”
+
+Asynchronous UpDown Counter
+
+* Name: totalHeapSize
+* Unit: “By”
+* Description: “The current total heap size”
+* Includes Callback Function
+
+UpDown Counter
+
+* Name: threadsActive
+* Unit: “1”
+* Description: “The total number of threads active”
+
+Asynchronous Gauge
+
+* Name: cpuUsage
+* Unit: “1”
+* Description: “Cpu usage percent”
+* Includes Callback Function
+
+**Common Attributes for Random based metrics**
+
+These are Key Value pairs to be added on metrics.
+```
+{
+“signal”:     (string)“metric”
+“language”:   (string)<name of language used>
+“metricType”: (string)“random”
+}
+```
+
+#### Logs
+
+**Logs are OPTIONAL due to their status of preview upstream. Logs must not be tested against nor will the logging structure described be final. It is to be used as an example to construct and emit example logs.**
+Logs will be displayed in JSON format for simplicity and readability.  Logs will be sent out on three different occasions, requests, errors, and events. An example of each can be seen underneath this section of the document. We will try to keep logs consistent in all sample applications according to the Logs Data Model (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md) found in the opentelemetry-specification repository.
+
+Logs will contain the following fields:
+
+* Timestamp
+* TraceID
+* SpanID
+* SeverityText (or LogLevel)
+* Body
+
+```
+Info Logs 
+
+// Info Logs are usually sent on application startup.
+
+{
+"level":"info",
+"msg":"Launching sampleapp: I am [sampleapp-svc] listening on port :8080 on all local IPs.",
+"time":"2022-06-28T11:31:38-07:00"
+}
+```
+
+```
+Event Log
+
+// Upon any event logs are sent out.
+
+{
+"event":"invoke","level":"info","msg":"sampleapp was invoked",
+"remote":"127.0.0.1:63875","time":"2022-06-27T15:58:35-07:00",
+"traceID":"1-62ba361b-0dc302c06b6bc21c2b2152a4"
+}
+```
+
+#### Traces
+
+At minimum, four GET requests endpoints must be present in each sample app. 
+The first endpoint will serve as a health check to confirm that the app has successfully ran.
```

Comment:
Let's move the description of the endpoints to a better position inline with the name of the endpoint